dash/testing/plugin.py

```diff
@@ -33,9 +35,13 @@ def __init__(self, **kwargs):
     DashComposite = MissingDashTesting
     DashRComposite = MissingDashTesting
     DashJuliaComposite = MissingDashTesting
+    _installed = False
```

Comment:
This will help with some cases, but it seems like some of the folks commenting on #946 actually have all the other packages installed separately, so the imports would succeed but they wish they didn't. I suppose we can go with it as is, just trying to figure out if there's some way we can do even better? I suppose we could make a dummy package like `dash-testing-stub` to add to `requires-testing.txt`, that does nothing but raise an `ImportError` if it's not there?

Reply:
Oh yea that won't be enough, the package stub should do the trick.